Author: Andreas Beckmann <anbe@debian.org>
Description: fix building with 64-bit time_t on 32-bit platforms

--- a/compiler/back-ends/c-gen/gen-code.c
+++ b/compiler/back-ends/c-gen/gen-code.c
@@ -485,7 +485,7 @@ PrintCSrcComment PARAMS ((src, m),
     FILE *src _AND_
     Module *m)
 {
-    long int t;
+    time_t t;
 
     t = time (0);
     fprintf (src, "/*\n");
@@ -589,7 +589,7 @@ PrintCHdrComment PARAMS ((f, m),
     FILE *f _AND_
     Module *m)
 {
-    long int t;
+    time_t t;
 
     t = time (0);
     fprintf (f, "/*\n");
--- a/compiler/back-ends/idl-gen/gen-code.c
+++ b/compiler/back-ends/idl-gen/gen-code.c
@@ -90,7 +90,7 @@ PrintComment PARAMS ((idl, m),
     FILE *idl _AND_
     Module *m)
 {
-    long t;
+    time_t t;
 
     t = time (0);
     fprintf (idl, "//   NOTE: this is a machine generated file -- editing not recommended\n");
